src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -57,10 +54,8 @@ public InventoryMock(@Nullable InventoryHolder holder, @NotNull InventoryType ty
 	 *
 	 * @param condition The condition to check for.
 	 */
-	public void assertTrueForAll(@NotNull Predicate<ItemStack> condition)
-	{
-		for (ItemStack item : items)
-		{
+	public void assertTrueForAll(@NotNull Predicate<ItemStack> condition) {
+		for (ItemStack item : items) {
```

Comment:
You broke the formatting here.
```suggestion
	public void assertTrueForAll(@NotNull Predicate<ItemStack> condition)
	{
		for (ItemStack item : items)
		{
```